### ISSUE TITLE
Amplify init fails on another machine

### ISSUE THREAD

**Issue by michaelcuneo** [aws-amplify/amplify-cli#1122]
I'm at work right... I type amplify init, I do all my work, later at home, I pull the project from github and want to work on it... so I hit amplify add env, pick the env that I added at work... 

It complains, fails to compile, won't do a thing... how am I meant to do this correctly?

** Which Category is your question related to? **
amplify in general... 

** What AWS Services are you utilizing? **
amplify-cli

** Provide additional details e.g. code snippets **

**Comment by kaustavghosh06** [comment]
What’s the error that you’re getting? Are the CLI versions the same as what you used at work and what you’re using at home? Would be helpful for us to know the error stack and get more information to root-cause your issue.

**Comment by michaelcuneo** [comment]
/home/michael/.nvm/versions/node/v8.15.0/lib/node_modules/@aws-amplify/cli/node_modules/gluegun/build/index.js:13
    throw up;
    ^

SyntaxError: Unexpected token h in JSON at position 0
    at JSON.parse (<anonymous>)

**Comment by kaustavghosh06** [comment]
Are the CLI versions the same as what you used at work and what you’re using at home? 
Also what are the files that you checked in in github? 
Can I take a look at your .gitignore file or the project structure when you pulled the project on your local machine at home?

**Comment by michaelcuneo** [comment]
I'm pretty sure they are. I regularly do a npm upgrade and update every single part of my project, on all sides. I git push at work, git pull at home. But amplify-js never appears to transfer across nicely.

```
# Don't check auto-generated stuff into git
coverage
build
node_modules
stats.json

# Cruft
.DS_Store
npm-debug.log
.idea

#amplify
amplify/\#current-cloud-backend
amplify/.config/local-*
amplify/backend/amplify-meta.json
aws-exports.js
awsconfiguration.json
```

That's my .gitignore, all autogenerated by things I install.

**Comment by kaustavghosh06** [comment]
@michaelcuneo Can you provide the file structure that you have in the system where you're seeing this issue?

**Comment by yuth** [comment]
@michaelcuneo Is one of the machine windows?

**Comment by michaelcuneo** [comment]
No, the other machine is an Intel server running Ubuntu. I use OSX at work, Ubuntu or Docker for my dev at home.

**Comment by yuth** [comment]
Released in Amplify CLI 1.6.6. Please re-open the issue if the problem still persist
